lib/live_school.ex

```diff
@@ -0,0 +1,137 @@
+#!/usr/bin/env elixir
+
+defmodule LiveSchool do
+  @moduledoc """
+  LiveSchool converts school lessons to LiveBook notebooks!
+
+  """
+
+  def rewrite_expre(code) do
+    wrap_code = fn x -> "\n```elixir\n" <> x <> "```\n" end
```

Comment:
Is there some benefit to establishing anonymous functions for `wrap_code/1` and `match_to_cell/1` over defining them as functions in this module?